api.go

```diff
@@ -103,8 +102,6 @@ func (c *cron) Delete(ctx context.Context, name string) error {
 	if _, err := c.client.Delete(ctx, jobKey); err != nil {
 		return err
 	}
-	counterKey := c.key.CounterKey(name)
```

Comment:
Why is this being removed? Won't this cause counters to linger even when a job is removed?